iso690-author-date-cs.csl

```diff
@@ -386,11 +386,11 @@
     <choose>
       <if variable="URL DOI" match="any">
         <group prefix=" [" suffix="]">
-          <text term="accessed" form="short"/>
+          <text term="accessed" form="short" suffix=".&#8239;"/>
           <date variable="accessed">
-            <date-part name="day" prefix=". "/>
-            <date-part name="month" prefix=". "/>
-            <date-part name="year" prefix=" "/>
+            <date-part name="year" suffix="-"/>
+            <date-part name="month" suffix="-" form="numeric-leading-zeros"/>
+            <date-part name="day" form="numeric-leading-zeros"/>
```

Comment:
I think you should use prefixes here. With suffixes, and y-m-d order, you would get "y-m-" and "y-" for partial dates, instead of the correct "y-m" and "y".